src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -0,0 +1,37 @@
+package lotto
+
+class LottoMachine(
+    val purchaseAmount: Int,
+    val tickets: MutableList<Lotto> = emptyList<Lotto>().toMutableList(),
+) {
+    private var change = 0
+    private var ticketCount = 0
+
+    init {
+        require(purchaseAmount in MIN..MAX)
+        change = purchaseAmount % TICKET_PRICE
+        ticketCount = (purchaseAmount - change) / TICKET_PRICE
+        generateTickets()
+    }
+
+    fun showChange() = change
+
+    fun generateTickets() {
+        val lottoTickets = List(ticketCount) { Lotto(generateNumbers()) }
+        tickets.addAll(lottoTickets)
+    }
+
+    private fun generateNumbers(): List<LottoNumber> =
+        numberList
+            .shuffled()
+            .take(Lotto.LOTTO_SIZE)
+
+    companion object {
+        private val numberList =
+            (LottoNumber.MINIMUM_NUMBER..LottoNumber.MAXIMUM_NUMBER)
+                .map { it -> LottoNumber.from(it) }
```

Comment:
Now the number range looks like static value.
Let's change the value name to uppercase! it'll be clearer for fellow developers.

Reply:
and it can also skip the `it` keyword 😄